rest/src/main/java/org/apache/unomi/rest/service/impl/RestServiceUtilsImpl.java

```diff
@@ -64,7 +64,7 @@ public String getProfileIdCookieValue(HttpServletRequest httpServletRequest) {
         if (cookies != null) {
             for (Cookie cookie : cookies) {
                 if (configSharingService.getProperty("profileIdCookieName").equals(cookie.getName())) {
-                    localBeanValidationProvider.get().validateBean(new CookieWrapper(cookie.getValue()));
```

Comment:
We can rename the variable to beanValidationService

Reply:
Thx I will rename it